crates/re_renderer/src/view_builder.rs

```diff
@@ -687,11 +712,54 @@ impl ViewBuilder {
         Ok(ScheduledScreenshot {
             row_info,
             identifier,
-            width: setup.resolution_in_pixel[0],
-            height: setup.resolution_in_pixel[1],
+            extent: setup.resolution_in_pixel.into(),
         })
     }
 
+    /// Schedules the readback of a rectangle from the picking layer.
+    ///
+    /// The result will still be valid if the rectangle is partially or fully outside of bounds.
+    /// Areas that are not overlapping with the primary target will be filled as-if the view's target was bigger.
+    ///
+    /// Note that the picking layer will not be created in the first place if this isn't called.
+    pub fn schedule_picking_readback(
+        &mut self,
+        ctx: &mut RenderContext,
+        picking_rect_min: glam::IVec2,
+        picking_rect_extent: u32,
+        show_debug_view: bool,
+    ) -> Result<ScheduledPickingRect, ViewBuilderError> {
+        if self.picking_processor.is_some() {
+            return Err(ViewBuilderError::PickingRectAlreadyScheduled);
```

Comment:
So users can only do one read per frame? That should probably be documented then!